hummingbot/core/data_type/trade_fills.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+from collections import namedtuple
+from typing import List
+from datetime import datetime
+import pandas as pd
+
+from hummingbot.core.event.events import (
+    TradeType,
+    TradeFee,
+    OrderType,
+)
+
+
+class TradeFills(namedtuple("_TradeFill", "symbol, side, price, amount, order_type, market, timestamp, trade_fee")):
+    symbol: str
+    trade_type: TradeType
+    price: float
+    amount: float
+    order_type: OrderType
+    market: str
+    timestamp: float
+    trade_fee: TradeFee
+
+    @classmethod
+    def to_pandas(cls, trades: List):
```

Comment:
This should be added to the `TradeFill` model class. A new data type class doesn't seem needed for now.